config/vufind/Blender.ini

```diff
@@ -0,0 +1,236 @@
+initialResults[] = EDS
+
+; The number of records to take from each backend when filling a result list in a
+; round-robin fashion:
+blockSize = 7
+
+; Block size based on result count (result count range[colon]block size):
+;adaptiveBlockSizes[] = "0-20:3"
+;adaptiveBlockSizes[] = "21-10000:5"
+
+; ---------- searches.ini settings ----------
+
+[General]
+default_handler = AllFields
+default_sort = "relevance,id asc"
+default_view = list
+default_limit = 20
+;limit_options        = 10,20,40,60,80,100
+case_sensitive_bools = true
+case_sensitive_ranges = true
+
+default_top_recommend[] = SpellingSuggestions
+;default_top_recommend[] = VisualFacets:Visual_Settings
+default_side_recommend[] = SideFacetsDeferred:Results:CheckboxFacets:Blender
+;default_noresults_recommend[] = SwitchTab
+default_noresults_recommend[] = SwitchType
+default_noresults_recommend[] = SwitchQuery:::fuzzy
+default_noresults_recommend[] = SpellingSuggestions
+default_noresults_recommend[] = RemoveFilters
+
+; Set this to true in order to highlight keywords from the search query when they
+; appear in fields displayed in search results.
+highlighting = true
+
+; Set this to restrict the list of fields that will be highlighted (the hl.fl
+; Solr parameter); default = '*' for all fields:
+;highlighting_fields = *
+snippets = true
+retain_filters_by_default = true
+;default_filters[] = "format:Book"
+;default_filters[] = "institution:MyInstitution"
+;default_filters[] = "(format:Book AND institution:MyInstitution)"
+
+; Whether the "versions" (FRBR) link and record tab are enabled. Default is true.
+;display_versions = true
+
+; This setting allows to limit pagination of a search result as deep pagination
+; costs a lot of performance and most users are not very likely to navigate
+; further down than 20 pages of a search result.
+; This is especially useful to prevent webcrawlers from descending too deep and
+; eating up search backend performance. Default is set to unlimited.
+; With Blender it is recommended to keep this as low as possible.
+result_limit = 400
+
+; This section controls the search handler options displayed in blended mode.
+[Basic_Searches]
+AllFields = "All Fields"
+Title = Title
+Author = Author
+Subject = Subject
+
+[Sorting]
+relevance = sort_relevance
+year = sort_year
+year asc = "sort_year asc"
+
+; ---------- facets.ini settings ----------
+
+; The order of display is as shown below
+; The name of the index field is on the left
+; The display name of the field is on the right
+[Results]
+; blender_backend is a pseudo-facet that returns result counts for each searched
+; backend:
+;blender_backend    = Source
+institution        = Institution
+building           = Library
+format             = Format
+author_facet       = Author
+language           = Language
+genre_facet        = Genre
+era_facet          = Era
+geographic_facet   = Region
+publishDate        = adv_search_year  ; share year string w/advanced search page
+; Authority facets:
```

Comment:
These are useful example fields, but I had completely forgotten that they even existed. Maybe we should add some sample records to the test data that populate these fields, so there's something to see when we test using the default index. (Obviously, that doesn't have to happen here, but might be a good separate PR).

Reply:
...maybe we can come up with a plan to do this as part of our authority data next steps conversation at next week's Community Call.

Reply:
These are copied from the default facets.ini. Agreed, adding more tests would be great, but out of scope here!